### ISSUE TITLE
please update for HAA and RavenCore v2) 12.0.2

### ISSUE THREAD

**Issue by aptyp222** [glumb/haa-configurator#44]
I really like your project, I use it all the time. I myself do not understand anything in programming, but thanks to you I can realize my ideas. Please update to version 12. https://github.com/RavenSystem/esp-homekit-devices It uses a slightly different algorithm system https://github.com/RavenSystem/esp-homekit-devices https://github.com/RavenSystem/esp-homekit-devices

**Comment by Hl2run** [comment]
Yes please, I am having really hard time migrating my configs. Thank you!

**Comment by l3zha** [comment]
Please take the time to update! Thanks :3

**Comment by Agargollo** [comment]
I have my project stopped because I can no longer configure my boards.
Update

**Comment by Bohan-Song** [comment]
**_It is very helpful to me, I hope to continue to maintain and update, I will be very grateful_**

**Comment by glumb** [comment]
Is there any migration guide or documentation regarding what changed?

**Comment by l3zha** [comment]
> Is there any migration guide or documentation regarding what changed?

https://github.com/RavenSystem/esp-homekit-devices/wiki/migration-guide-for-haa-v12-merlin

**Comment by Rahulsharma0810** [comment]
Really helpful tool, Please Migrate to the latest version.

**Comment by shengweitw** [comment]
> 是否有相關更改的遷移指南或文檔？

I like this very much, it's very useful and I'm looking forward to the new version.

**Comment by sorriso93** [comment]
If update is planned it would be an excellent news...

**Comment by Agargollo** [comment]
Atualize para a versão 12. https://github.com/RavenSystem/esp-homekit-devices Ele usa um sistema de algoritmo ligeiramente diferente [https://github.com/RavenSystem/esp-homekit-devices ](https://github.com/RavenSystem/esp-homekit-devices)[https://github.com /RavenSystem/esp-homekit-devices](https://github.com/RavenSystem/esp-homekit-devices)
